.github/workflows/integration.yml

```diff
@@ -0,0 +1,95 @@
+name: Integration
```

Comment:
I think this is more of an `e2e` test than an `integration` one.

Reply:
renamed